core/trino-main/src/test/java/io/trino/server/security/oauth2/TestOAuth2WebUiAuthenticationFilter.java

```diff
@@ -140,13 +141,15 @@ public void setup()
                         .put("http-server.https.enabled", "true")
                         .put("http-server.https.keystore.path", Resources.getResource("cert/localhost.pem").getPath())
                         .put("http-server.https.keystore.key", "")
-                        .put("http-server.authentication.oauth2.auth-url", "http://hydra:4444/oauth2/auth")
```

Comment:
Should we have some tests for plain http?

Reply:
Would anyone ever run an OAuth service over plain HTTP?